modules/ROOT/pages/customsidebar.adoc

```diff
@@ -6,6 +6,8 @@
 
 {productname} allows developers to create sidebars and add custom UI widgets inside a constrained and easily accessible area of the editor. The sidebar is designed to allow administrators and plugin developers to provide additional tools that can be accessed by {productname} users.
 
+include::partial$misc/admon-iframe-only.adoc[]
```

Comment:
This partial uses the word option, where in this case a `sidebar` isn't an option, but a UI component which can be added through the `editor.ui.registry.addSidebar` api. I suggested that this partial be included under the `sidebar_show` option instead, but if it were to be included here then it may need to be changed to reflect the type of component used by the page calling it. Might be a bit too complicated though.

Reply:
Corrected with [this commit](https://github.com/tinymce/tinymce-docs/pull/2540/commits/54b1f11b5b880997eef491d3fb504630c7a29f1c).